CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changlog for Omnitrace
+
+Full documentation for Omnitrace is available at
+[https://rocm.docs.amd.com/projects/omnitrace/en/latest](https://rocm.docs.amd.com/projects/omnitrace/en/latest).
+
+## Omnitrace v1.11.2 for rocm-6.2.0
```

Comment:
```suggestion
## Omnitrace v1.11.2 for ROCm 6.2.0
```
The particular formatting might cause issues with https://github.com/ROCm/ROCm/tree/develop/tools/autotag

Also, should this be `6.2.1` or `6.2.0`?